### DISCUSSION TITLE
Better Sorting

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#296] [Feature Requests]
## Nota Currently
Currently, Nota does offer a way to sort your notes by Name, Modified, or Date Created like so:
![CleanShot 2022-08-22 at 00 28 58@2x](https://user-images.githubusercontent.com/48145861/185840166-61083354-7a7b-47e3-87f8-94d2828b6827.png)

## Apple Notes Example
However, it's still limited as there is no option to choose if it's ascending or descending. I'd love to have the option to choose newest to oldest and vice versa, and A-Z or Z-A for the name option as shown in Apple Notes:
![CleanShot 2022-08-22 at 00 31 36@2x](https://user-images.githubusercontent.com/48145861/185840354-6c4588af-2643-433f-ba6f-e6eee512d994.png)
![CleanShot 2022-08-22 at 00 31 58@2x](https://user-images.githubusercontent.com/48145861/185840359-afe5a05c-5057-4728-af77-7f3a7217fc9f.png)

## More Info
The reason this is important is that if one wants to add a note for a specific day, they can title the note and the name of the file like: 2022-08-12 for example, and Nota should know exactly how to sort the files especially if someone wants the most recently created notes on top or vice versa.

That's it. Thank you!

**Comment by astoilkov** [comment]
This is a planned feature. We will improve on this very soon. Thanks!

**Reply by marlonjames71** [reply]
Awesome. I look forward to it. Thank you!
